### ISSUE TITLE
Google Play Store rejects app because of this package 

### ISSUE THREAD

**Issue by lucaesposto** [Mixpeal/flutter_paypal#15]
Google Play Store rejects app because some dependencies of this package.

This is an excerpt of error message:
**Issue with your app**
We found that your app is using Tencent TBS SDK which contains code to download executable code (e.g. dex, JAR, .so files) from a source other than Google Play.
If necessary, you can consult your SDK provider(s) for further information.

**About the Device and Network Abuse policy**
An app distributed via Google Play may not modify, replace, or update itself using any method other than Google Play's update mechanism. Likewise, an app may not download executable code (e.g. dex, JAR, .so files) from a source other than Google Play. This restriction does not apply to code that runs in a virtual machine or an interpreter where either provides indirect access to Android APIs (such as JavaScript in a webview or browser).

Apps or third-party code (e.g., SDKs) with interpreted languages (JavaScript, Python, Lua, etc.) loaded at run time (e.g., not packaged with the app) must not allow potential violations of Google Play policies.
![image](https://user-images.githubusercontent.com/84773245/233138758-ce9404ae-ec9c-4833-ad48-b231492fd9ce.png)

**Comment by DaenPve** [comment]
Yeah i have the same issue, but there is nothing that you can do, because webview use TENCENT TBS SDK, i hope a fix to that.

**Comment by Mixpeal** [comment]
Hello, I updated the webview some days ago, are you sure you have latest version?

**Comment by hardiklakhalani** [comment]
> updated the webview some days ago

@Mixpeal  Could you please mention this commit so we can know what changes has been made?

**Comment by rvharjinderbains** [comment]
I am also getting the same issue. I have removed this package and app is approved. Please fix this issue
![Screenshot 2023-10-06 131951](https://github.com/Mixpeal/flutter_paypal/assets/105272009/aa7af5a1-067e-4f30-b977-6514ba309e69)
